script/1_0_2current_en_script/Common/00_02_hidden.txt

```diff
@@ -1,14 +1,14 @@
 <Page3>
 たとえば,
-For example
+For example,
 もし、空が雲っていたのなら.
 What if, the sky had been cloudy.
 もし, 気づくのが少し早ければ.
-What if, I had noticed it a little sooner.
+What if, I had noticed a little sooner.
 もし, 彼がおとろえていなければ.
-What if, he hadn't weakened.
+What if, he wasn't weakened.
```

Comment:
I think this was correct before (I believe JP is "what if he had not become weak", rather than "was not [already] weak")